assets/js/renderer.js

```diff
@@ -410,17 +415,7 @@ $(document).ready(function () {
 
     // Open a SQLite Database File -- custom menu
     ipcRenderer.on('open-SQLite', function () {
-        dialog.showOpenDialog({
-            properties: ['openFile'],
-            title: "Open a SQLite file",
-            filters: [{
-                name: 'SQLite File',
-                extensions: ['sqlite']
-            }]
-        }, function(fileNames) {
-            if (fileNames === undefined) return;
-            loadDatabaseFromFile(fileNames[0]);
-        });
+        openSQLite();
```

Comment:
Same comment as above. 

Also can you move this code to right below the other call (`openSQLiteFileButton.click`). This will make it easier for someone reading the code to see all of the usages.